api/prisma/seed-dev.ts

```diff
@@ -84,21 +84,21 @@ export const devSeeding = async (
 
   await reservedCommunityTypeFactoryAll(jurisdiction.id, prismaClient);
 
-  const householdSize = randomInt(0, 6);
   for (let index = 0; index < LISTINGS_TO_SEED; index++) {
-    const householdMembers = await householdMemberFactoryMany(householdSize);
-
     const applications = [];
 
-    for (let j = 0; j <= APPLICATIONS_PER_LISTINGS; j++) {
-      applications.push(
-        await applicationFactory({
-          householdSize,
-          unitTypeId: unitTypes[randomInt(0, 5)].id,
-          householdMember: householdMembers,
-          multiselectQuestions,
-        }),
+    for (let j = 0; j <= APPLICATIONS_PER_LISTINGS - 1; j++) {
```

Comment:
Previously, all applications had the same household size and same household members